code/api/src/main/scala/com/avast/datadog4s/api/metric/Timer.scala

```diff
@@ -1,11 +1,14 @@
 package com.avast.datadog4s.api.metric
 
 import java.time.Duration
+import scala.concurrent.duration.FiniteDuration
 
 import com.avast.datadog4s.api.Tag
 
 trait Timer[F[_]] {
   def time[A](f: F[A], tags: Tag*): F[A]
 
-  def record(duration: Duration, tags: Tag*): F[Unit]
+  def record(duration: Duration, tags: Tag*): F[Unit]       = recordMillis(duration.toMillis, tags: _*)
```

Comment:
I'm not really a big fan of method overloading as it makes it much harder to pass method as value, ie. `timer.record _` can't really be used.

We could also make the method generic and require some sort of typeclass instance for that type. NOt sure if it's worth though.

Reply:
i decided to add the typeclass, no need to limit it to just couple of certain types